src/deadline/max_adaptor/MaxClient/render_handlers/vray_handler.py

```diff
@@ -0,0 +1,64 @@
+"""
+3ds Max Deadline Cloud Adaptor - V-Ray specific actions
+
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+"""
+
+import sys
+
+import pymxs  # noqa
+from pymxs import runtime as rt
+
+from .default_max_handler import DefaultMaxHandler
+
+# Re-assign sys stdout and stderr to print in the console instead of the Max Listener
+sys.stdout = sys.__stdout__
+sys.stderr = sys.__stderr__
+
+
+class VrayHandler(DefaultMaxHandler):
+    """Render Handler for V-Ray"""
+
+    def __init__(self, GPU):
+        """
+        Initializes the V-Ray and V-Ray Handler
+        """
+        super().__init__()
+        self.GPU: bool = GPU
```

Comment:
One of the code scanners is flagging this variable as not following conventions. Can the variable and input parameter be changed to all lowercase `gpu`?